packages/features/src/categories/error-tracking/patterns/logrocket.ts

```diff
@@ -0,0 +1,67 @@
+import { Page } from 'playwright';
+import { AnalysisFeatures } from '../../../types/analysis.js';
+
+export const logrocket = [
+  {
+    name: 'globals' as const,
+    score: 1.5,
+    browser: async (page: Page) => {
+      return page.evaluate(() => {
+        const markers = {
+          hasLogRocket: typeof window.LogRocket !== 'undefined',
+          hasLogRocketConfig:
+            typeof window.__LOGROCKET_CONFIG__ !== 'undefined',
+          hasLRQueue: Array.isArray(window._LRQueue),
+        };
+        return Object.values(markers).some(Boolean);
+      });
+    },
+  },
+  {
+    name: 'elements' as const,
+    score: 1.2,
+    browser: async (page: Page) => {
+      return page.evaluate(() => {
+        const markers = {
+          hasLogRocketScripts: Array.from(
+            document.querySelectorAll('script')
+          ).some(
+            (script) =>
+              script.src &&
+              (script.src.includes('cdn.logrocket.') ||
+                script.src.includes('cdn.lr-') ||
+                script.src.includes('logrocket@'))
+          ),
+        };
+        return Object.values(markers).some(Boolean);
+      });
+    },
+  },
+  {
+    name: 'core' as const,
+    score: 1.2,
+    scripts: [
```

Comment:
We can add more detections. For example, check minified code: https://staging.logrocket.com/LogRocket.min.js
it does contai a lot of markers which survive minification:
- strings (`" ... LogRocket truncating to first 1000 characters.\n      Keep data under 4MB to prevent truncation. https://docs.logrocket.com/reference/network"`, `"LogRocket skipped consuming an event-stream body."`, `"LogRocket error reading body: "`, `"LogRocket: Error accessing response."}`, etc)